ppb/engine.py

```diff
@@ -8,6 +8,19 @@
 class GameEngine(Engine):
 
```

Comment:
Missing class docstring.

Reply:
I've changed my mind on this. I won't block this PR based on a docstring missing.